daphne_service_utils/src/durable_requests/mod.rs

```diff
@@ -323,4 +323,23 @@ mod test {
         let got = DurableRequest::try_from(&req_bytes).unwrap();
         assert_eq!(want, got);
     }
+
+    // We use `std::fmt::Display` for `DapBatchBucket` to format names for DO instances. Ensure
+    // that they are formatted the way we expect.
+    #[test]
+    fn bucket_display() {
+        assert_eq!(
+            "batch/1111111111111111111111111111111111111111111111111111111111111111",
+            format!(
+                "{}",
+                DapBatchBucket::FixedSize {
+                    batch_id: BatchId([17; 32])
+                }
+            )
+        );
+        assert_eq!(
+            "window/1337",
+            format!("{}", DapBatchBucket::TimeInterval { batch_window: 1337 })
+        );
+    }
```

Comment:
Why put this test in this file instead of the `bindings.rs` file? Since the code that actually uses this display implementation is there

Reply:
Good idea, moved.